### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by 111100001** [sonigy/YCS#11] [Ideas]
I love your work so much!!

I would really like it if you could make it possible to load all the comments on a whole channel rather than just one video.

**Comment by Letssharearow** [comment]
that's a fun Idea, but I believe that would take a while to process. Technically it shouldn't be too hard to implement though, right? It could cause a lot of overhead that's why it needs to work very well and would probably take a while to test

**Comment by 111100001** [comment]
this already exists but i don't like the sorting system and the timeline of replys https://ytcomment.kmcat.uk/

**Reply by Letssharearow** [reply]
wow, this is pretty cool though, but you're right. Being able to search in more detail wouldnt cause any harm
